Woosmap/geofencing-core-android-sdk
# New method `getUserPropertyMap` to get POI custom properties

## Issue
closes #19 

## Describe your changes
1. Exposed new method `getUserPropertyMap`
2. Saving only feature JSON in `POI.data` property and not the whole of response.

## How to test
1. Log on to the Woosmap console and Modify your assets to add custom properties. 
2. Fetch the POIs using SDK and check if `getUserPropertyMap` returns the custom properties.

## Checklist:



## Comments
